components/molecules/InputForm.tsx

```diff
@@ -0,0 +1,121 @@
+import { Eye, EyeOff, LockKeyhole, Mail, Phone, User2 } from '@tamagui/lucide-icons'
+import { isNil } from 'lodash'
+import React, { useState } from 'react'
+import { useColorScheme } from 'react-native'
+import { Image, Text, View, YStack } from 'tamagui'
+
+import InputWithIcons from '~/components/atoms/InputWithIcons'
+import { NegativeButton } from '~/components/atoms/NegativeButton'
+import { PositiveButton } from '~/components/atoms/PositiveButton'
+import { TextTitle } from '~/components/atoms/TextTitle'
+import getColors from '~/constants/Colors'
+import useTranslation from '~/hooks/useTranslation'
+
+interface Props {
```

Comment:
Use `type` instead of `interfaces`, it look like 

```suggestion
type Props =  {
 ....
} & InputProps
```